checkov/terraform/module_loading/loaders/git_loader.py

```diff
@@ -75,7 +75,7 @@ def _parse_module_source(self) -> ModuleSource:
 
         username = re.match(re.compile(r"^(.*?@).*"), root_module)
         if username:
```

Comment:
```suggestion
        if username and username[1] != "git@":
```
why not use the if block here 😉

Reply:
You're right, as always